source/includes/administration/_rollbacks.md

```diff
@@ -153,17 +157,25 @@ curl "https://cloudmc_endpoint/rest/rollbacks/23910576-d29f-4c14-b663-31d728ff49
 
 `POST /rollbacks`
 
-Creates a rollback which will result in reprocessing of usage for the given request body. 
-This API currently supports the REPROCESS rollback type which uses existing usage previously collected by plugin usage collectors and re processes it in the monetization engine. The latest pricing configuration will be used when processing the records. 
+Creates a rollback which will result in reprocessing of usage for the given request body. The Rollback API supports two types of rollback. `REPROCESS` and `RECOLLECT`
 
-The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin. 
-
-> Note: You must have the Reseller pricing permission on the target reseller. If the caller does not have the correct permissions or the reseller with the ID provided does not exist then a `404 Not Found` response will be returned.
+The `REPROCESS` rollback type uses existing usage that was previously collected from a backend service and re-processes it in the monetization engine. The latest pricing configuration will be used when these records. 
```

Comment:
"uses existing usage" sounds a bit off. There's also a typo in the 2nd sentence right?

```suggestion
The `REPROCESS` rollback re-processes existing usage, previously collected from the backend service, in the monetization engine. The latest pricing configuration will be used on these records. 
```

Reply:
Thanks... Sorry I missed that.